packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -270,6 +270,14 @@ class AnimationController extends Animation<double>
   /// The most recently returned [TickerFuture], if any, is marked as having been
   /// canceled, meaning the future never completes and its [TickerFuture.orCancel]
   /// derivative future completes with a [TickerCanceled] error.
+  ///
+  /// During the animation, [status] is reported as [AnimationStatus.forward],
+  /// which switches to [AnimationStatus.completed] when [upperBound] is
+  /// reached at the end of the animation.
+  ///
+  /// You should not call [forward] or [reverse] on the same instance of an
+  /// [AnimationController] on which you also call [animateTo]. If you do mix
+  /// those calls, [status] might not have the expected value.
```

Comment:
i'd say "might not have the value you expect", rather than _the_ expected value.